{"Nodes":[],"ExecutionTime":"2024-10-01T16:26:58.2000999Z","GenerationTime":"0001-01-01T00:00:00","PluginUserSpecFlowId":"cee11188-27b4-400a-9715-1dbf4ebdd42e","CLIUserSpecFlowId":null,"ExecutionResults":[{"ContextType":"Scenario","FeatureFolderPath":"Features","FeatureTitle":"LoginPage","ScenarioTitle":"Fluxo de Login","ScenarioArguments":[],"Status":"TestError","StepResults":[{"Duration":"00:00:11.7729450","Status":"OK","Error":null,"Outputs":null},{"Duration":"00:00:08.2515965","Status":"TestError","Error":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:57157/session/17faa1dfec1f4759d58df7a6e7220be2/element. The exception message was: An error occurred while sending the request.","Outputs":null}],"Outputs":null}],"StepReports":null}